awswrangler/s3/_read_parquet.py

```diff
@@ -382,7 +384,7 @@ def read_parquet(
     version_id: Optional[Union[str, Dict[str, str]]] = None,
     chunked: Union[bool, int] = False,
     use_threads: Union[bool, int] = True,
-    parallelism: int = 200,
+    parallelism: int = -1,
```

Comment:
Nice, could we change the method docs below to reflect that? It's saying 200 by default at the moment

Reply:
Yep, will do